packages/chargebee/src/checkout/index.tsx

```diff
@@ -22,22 +30,13 @@ const Checkout = ({
       success: () => { isSuccess = true; },
       error: onError,
       close: () => {
-        invoke(onClose);
```

Comment:
`invoke(onClose)` is not functionally equal to `onClose?.()`

Wouldn't do a breaking release because of it but just watch out when testing in apps

Reply:
Good point. I would just a `.call` instead of just invoking. That should replace the `is function` guard clause.